app/operations/subject_groups/create.rb

```diff
@@ -83,12 +96,32 @@ def build_group_subject
 
     def subject_location_params
       subject_locations = find_subjects_in_order(subject_ids).map(&:locations).flatten
+      external_locations_subject_id_lut = {}
```

Comment:
What does `lut` stand for in this variable name?

Reply:
Look Up Table (LUT) - it's a pretty big variable name already but can certainly be modified if you prefer?

Reply:
Maybe a comment next to the variable name would be helpful! e.g. `# look up table`